docs/components/email/index.html

```diff
@@ -0,0 +1,159 @@
+---
+title: Email
+minver: 0.20.0
+also:
+  components/email/test.html: Testing - Email
+  elements/hx-email-control/index.html: <hx-email-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+<p>
+  The {{page.title}} component provides supplmental markup to define
+  and style an email address input control.
+</p>
+{% endblock %}
+
+{% block content %}
+<section>
+  <header>
+    <h2 id="basic-text-input">Basic Email Input</h2>
```

Comment:
Please change id
like this : id="basic-email-input" or id="basic-email"

Reply:
Changed to `id="basic-email-input"`.